lib/assessment.js

```diff
@@ -32,14 +36,20 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
-
+  newArray = [];
+  if (array.forEach(callback) === true) {
+    newArray = array.map(callback);
+  }
+  return newArray;
```

Comment:
You've got a lot of the main elements, but you haven't yet plugged them together. 

Conceptually, what you want to do is
a) create a new array
b) iterate through every element in the old array and
   i) if the element causes the callback to return true, add it to the new array
c) once finished with the iteration, return the new array.

Think you can implement this one now?